ui/tests/acceptance/enterprise-transform-test.js

```diff
@@ -216,7 +216,7 @@ module('Acceptance | Enterprise | Transform secrets', function (hooks) {
     assert.dom('[data-test-row-value="Allowed roles"]').hasText(roleName);
     // Edit transformation
     await click('[data-test-edit-link]');
-    assert.dom('.modal.is-active').exists('Confirmation modal appears');
+    assert.dom('.hds-modal#transformation-edit-modal').exists('Confirmation modal appears');
```

Comment:
We shouldn't need the class as a selector since the ID should be unique. If you want to also check that it's an hds modal, you could do: 
```suggestion
    assert.dom('#transformation-edit-modal').hasClass('hds-modal', 'Confirmation modal appears');
```

Reply:
Ah, good catch. I was using the pattern already established. Should I go ahead and remove the class from the other instances to keep it consistent?

Reply:
In the other instances I included `.hds-modal` just so it was clear it was a modal - but I suppose in the id's I added I included `-modal` in most (all?) of them - feel free to update!